resources/lang/de/PaymentMethod.properties

```diff
@@ -1 +1,2 @@
-paymentMethodName = "Barzahlung"
\ No newline at end of file
+paymentMethodName = "Barzahlung"
+paymentMethodDescription = ""
```

Comment:
Brauchen wir hier keine Übersetzung?

Reply:
Über `paymentMethodDescription` wird die Beschreibung der Zahlungsart in CMS -> Mehrsprachigkeit geregelt. Die Beschreibung soll standardmäßig nicht gesetzt sein, deswegen ist die Variable leer.